src/libdrakvuf/vmi.c

```diff
@@ -1121,6 +1160,13 @@ bool inject_trap_pa(drakvuf_t drakvuf,
             PRINT_DEBUG("%s: Failed to change gfn on view %u\n", __FUNCTION__, drakvuf->altp2m_idr);
             goto err_exit;
         }
+
+        if (VMI_FAILURE == vmi_slat_change_gfn(
+        drakvuf->vmi, drakvuf->altp2m_idrx, remapped_gfn->r, drakvuf->sink_page_gfn))
```

Comment:
I don't see the `sink_page_gfn` getting its `rwx` permissions removed in the `rx` view.